CountlyTests/CountlyTests.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  CountlyTests.swift
+//  CountlyTests
+//
+//  Created by Muhammad Junaid Akram on 22/12/2023.
+//  Copyright © 2023 Alin Radut. All rights reserved.
+//
+
+import XCTest
+@testable import Countly
+
+
+class CountlyTests: CountlyBaseTestCase {
+    
+    // MARK: - Configuration Tests
+    
+    func testReInitWithDeviceId() throws {
```

Comment:
this test case testing multiple scopes like halting, there will be probably a test case about it. I wanted to indicate in case